presto-native-execution/presto_cpp/main/PrestoTask.cpp

```diff
@@ -561,7 +561,7 @@ void PrestoTask::updateMemoryInfoLocked(
   const double sinceLastPeriodMs = currentTimeMs - lastTaskStatsUpdateMs;
 
   prestoTaskStats.cumulativeUserMemory +=
-      (averageMemoryForLastPeriod * sinceLastPeriodMs) / 1000;
+      (averageMemoryForLastPeriod * sinceLastPeriodMs);
```

Comment:
@xiaoxmeng can you clarify why this change is required? Thanks.

Reply:
IIRC, java use ms instead of us. The change is to align with java. without this change, native reports much less accumulative memory than java.